packages/flutter/test/material/chip_test.dart

```diff
@@ -1484,7 +1484,11 @@ void main() {
 
     final ChipThemeData chipTheme = theme.chipTheme;
 
-    InputChip inputChip = const InputChip(label: Text('Label'), pressElevation: 8.0);
+    InputChip inputChip = const InputChip(
+      label: Text('Label'),
+      pressElevation: 8.0,
```

Comment:
We should test that when `elevation` and `pressElevation` are not supplied, the defaults are `0.0` and `8.0` respectively

Reply:
Good idea, done